src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/data/GraphQLEntrypoint.java

```diff
@@ -0,0 +1,25 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data;
+
+import java.util.Map;
+
+/**
+ * Used to represent and contain information about a specific field
+ * in the GraphQL query type
+ */
+public interface GraphQLEntrypoint {
+    /**
+     * @return the GraphQL query fieldname
+     */
+    public String getFieldName();
+
+    /**
+     * @return the parameter definitions for the entrypoint as a map
+     * consisting of parameter names mapped to parameter value types.
+     */
+    public Map<String,String> getParameterDefinitions();
+
+    /**
+     * @return the GraphQL type the entrypoint represents
+     */
+    public String getType();
```

Comment:
Is the type a string? Shouldn't there be a Java class (or interface) whose instances represents such types? (I mean, for other kinds of elements of GraphQL schemas you also have Java classes/interface; for instance, this very interface here.)